pages/linux/sic.md

```diff
@@ -0,0 +1,25 @@
+# sic
+
+> Simple IRC client.
+> Part of the suckless tools.
+> More information: <https://tools.suckless.org/sic/>.
+
+- Connect to a host with the specified nickname:
+
+`sic -h {{irc.ofct.net}} -n {{$USER}}`
```

Comment:
Maybe we could add the most basic example before this one?

```markdown
- Connect to the default host (irc.ofct.net) with the nickname set in the `$USER` variable:

`sic`
```

Then this one could be simply:

```markdown
- Connect to a host using a given nickname:

`sic -h {{host}} -n {{nickname}}`
```

Reply:
I like, if you submit the suggestion, i'll add it.

Reply:
WDYT @bl-ue?

```suggestion
- Connect to the default host (irc.ofct.net) with the nickname set in the `$USER` variable:

`sic`

- Connect to a given host, using a given nickname:

`sic -h {{host}} -n {{nickname}}`
```

Reply:
Looks good to me, one thing though: ~`$USER` variable~ → **`$USER` environment variable**, right, @waldyrious?

Reply:
Right. I pondered that, and chose not to make the description too long, but it's best to be explicit. 👍